python/pyarrow/rados.py

```diff
@@ -24,12 +24,33 @@
 
 
 class SplittedParquetWriter(object):
+    """
+    Write data to a file of RadosParquetFile format.
```

Comment:
This does not write to a single file of "RadosParquetFileFormat". We can say "Split a large multi rowgroup Parquet file to a set of single row group Parquet files of a specified size."

Reply:
Done.